aws/src/main/java/org/apache/iceberg/aws/s3/S3FileIO.java

```diff
@@ -68,6 +68,7 @@
   private MetricsContext metrics = MetricsContext.nullMetrics();
   private final AtomicBoolean isResourceClosed = new AtomicBoolean(false);
   private Set<Tag> writeTags = Sets.newHashSet();
+  private Map<String, String> bucketToAccessPointMapping = Maps.newHashMap();
```

Comment:
nit: variable here can just be named `accessPoints`

Reply:
I agree with this,  but by naming it accessPoints at first glance could mean just a collection of access points where as here what we actually have  is a mapping from s3 bucket to accessPoint, if this rational makes sense I would like to keep it as its.. WDYT ?

Reply:
yes good point, let's keep as is